src/PKSim.Core/Snapshots/Mappers/MoleculeMapper.cs

```diff
@@ -83,6 +83,11 @@ public override async Task<IndividualMolecule> MapToModel(Molecule snapshot, Mod
          };
 
          await _expressionContainerMapper.MapToModels(snapshot.Expression, context);
+
+         //We need to normalize relative expressions when loading from old format
+         if (isV9Format(snapshot))
+            NormalizeRelativeExpressionCommand.NormalizeExpressions(individual, molecule);
```

Comment:
Normalizing if v9 format. Hopefully, we won't have too many of those conversion steps in the future because the code is going to become a nightmare